test/SlowTests/Issues/RavenDB_18938.cs

```diff
@@ -0,0 +1,378 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.NotificationCenter.Notifications;
+using Sparrow.Json.Parsing;
+using Sparrow.Server.Collections;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_18938 : RavenTestBase
```

Comment:
Would it alert on the index from https://issues.hibernatingrhinos.com/issue/RavenDB-15476 as well?

Reply:
No. It has a depth about `this21.[..].this0` but still SOE happens.